API/Data/Repositories/ReadingListRepository.cs

```diff
@@ -185,10 +186,11 @@ public void BulkRemove(IEnumerable<ReadingListItem> items)
 
     public async Task<PagedList<ReadingListDto>> GetReadingListDtosForUserAsync(int userId, bool includePromoted, UserParams userParams, bool sortByLastModified = true)
     {
-        var userAgeRating = (await _context.AppUser.SingleAsync(u => u.Id == userId)).AgeRestriction;
+        var user = await _context.AppUser.SingleAsync(u => u.Id == userId);
```

Comment:
Use first instead. Single does an additional check that there is only one, which is overhead we don't need.